style.css

```diff
@@ -630,6 +732,98 @@ main {
     overflow-x: hidden !important;
   }
 
+  .project_details {
+    position: fixed;
+    top: 0;
+    right: 0;
+    left: 0;
+    bottom: 0;
+    margin: auto;
+    width: 80%;
+    height: 95%;
+    background-color: #fff;
```

Comment:
- [ ] You did an amazing job adding the style to the pop-up but I suggest you to stick as much as you can to [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122) as it was of the project requirements. 

Here you have a screenshot of your modal:

![image](https://user-images.githubusercontent.com/74506933/156667433-9b63cd60-b38d-4236-afce-9a3c32bf7ba8.png)

Here you have a screenshot from the design:

![image](https://user-images.githubusercontent.com/74506933/156667513-70993ab7-8ebb-4012-98ac-250a02f3b5ab.png)